src/libs/language-python/api-parser/tests/modules/one_dawson_function.py

```diff
@@ -0,0 +1,10 @@
+""" A module with a dawson-valid Python function """
+
+def myfunction(event, context):
+    """ This function is dawson-valid """
+    print(event, context)
+    return "Hello world"
+
+myfunction.api = {
+    "path": "/test"
```

Comment:
small change needed here: path should not start with a slash (see https://dawson.sh/docs.html#4-function-configuration)